app.js

```diff
@@ -6,11 +6,15 @@ var sla = require('./lib/sla.js');
 var Q = require('q');
 
 var connect = Q.denodeify(ari.connect);
-
-connect('http://127.0.0.1:8088', 'user', 'pass')
-  .then(clientLoaded)
-  .catch(errHandler)
-  .done();
+var confFile;
+if (confFile = process.argv[2]) {
+  connect('http://127.0.0.1:8088', 'user', 'pass')
+    .then(clientLoaded)
+    .catch(errHandler)
+    .done();
+} else {
+  console.error('No configuration file specified');
```

Comment:
Is it fatal to not provide or find a configuration file? I would imagine this should actually bail out of the application.